src/main/java/frc/utils/controllers/joysticks/SmartJoystick.java

```diff
@@ -1,12 +1,12 @@
-package frc.utils.joysticks;
+package frc.utils.controllers.joysticks;
 
 import edu.wpi.first.math.MathUtil;
 import edu.wpi.first.wpilibj.GenericHID;
 import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj2.command.button.JoystickButton;
 import edu.wpi.first.wpilibj2.command.button.POVButton;
 
-public class SmartJoystick {
+public class SmartJoystick{
```

Comment:
What's wrong with the space, return it please😊(use ctrl + alt + L, or help me find solution to auto format)